estate/views/estate_menus.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estate">
+        <menuitem id="estate_menu_properties" name="Properties">
+            <menuitem id="estate_property_menu_action" action="estate_property_action"/>
```

Comment:
```suggestion
            <menuitem id="action_estate_property_menu" action="action_estate_property"/>
```
Action names must start with `action_` followed by what it performs. Please check all the implementation of actions

Reply:
done